<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellDialog</class>
 <widget class="QDialog" name="SellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>111</y>
      <width>500</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>264</x>
       <y>260</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QComboBox" name="assetComboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>73</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>70</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(192, 196, 212);</string>
     </property>
     <property name="text">
      <string>换成</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="assetComboBox2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>73</y>
       <width>90</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>卖数量</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>147</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>买数量</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="sellAmountLineEdit">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>113</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="buyAmountLineEdit">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>150</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>当前账户</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="accountNameLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>472</x>
       <y>10</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>184</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>输入钱包密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwdLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>187</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>224</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>71</y>
       <width>181</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>271</x>
       <y>70</y>
       <width>20</width>
       <height>151</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="depositBtn">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(84,116,235);
font: 10px &quot;微软雅黑&quot;;
border:none;
background:transparent;</string>
     </property>
     <property name="text">
      <string>合约里没有余额？前往充值</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
